tasks/configuration.yml

```diff
@@ -0,0 +1,62 @@
+---
+- name: Create configuration directories
+  file:
+    path: "{{ item }}"
+    state: directory
+    owner: root
+    group: "{{ icingaweb2_unix_group }}"
+    mode: 0755
+  with_items:
+    - /etc/icingaweb2/enabledModules
+    - /etc/icingaweb2/modules/monitoring
+
+- name: Create configuration files
+  template:
+    src: "{{ item }}.ini.j2"
+    dest: "/etc/icingaweb2/{{ item }}.ini"
+    owner: root
+    group: "{{ icingaweb2_unix_group }}"
+    mode: 0640
+  with_items:
+    - authentication
+    - config
+    - groups
+    - resources
+    - roles
+
+- name: Enable modules
+  file:
+    src: "{{ icingaweb2_module_path.split(':') | first }}/{{ item }}"
```

Comment:
Why are you using : to split

Reply:
Because that's how you specify the path for icingaweb2. It's their style, not mine